spam-n-eggs/ansible-aws-transfer
# PG-560: Updated so adding an existing user doesn't error out.

Details
===
- Eliminates errors when a user exists.
- Changed Namespace to reflect created namespaces.

Steps to test changes
===
1. Setup an args.json with the following arguments:
*name:* an SFTP server name, such as `sftp.mydomain.com`
*state:* add_user
*aws_access_key:* An AWS access Key
*aws_secret_key:* The Secret Key to go with the access Key
*region:* The Region the SFTP Server lives in
*vpc_id*: The VPC the SFTP Server lives in
*user_name:* A username to use
*user_home_directory:* a S3 path
*user_home_directory_type:* "PATH",
*user_role*: the ARN for a user role to use
1. Run the following line of code python3 ansible_collections/spam_n_eggs/amazon/plugins/modules/transfer.py args.json
1. Manually verify that the user has been created on the specified AWS Transfer User.
1. Re-run the python3 call from above.
1. Verify no errors have been thrown.

Issues Addressed
===
[PG-560](https://gfm-it.atlassian.net/browse/PG-560)
